CHANGELOG.md

```diff
@@ -4,6 +4,209 @@
 -->
 # Changelog of FOSSology
 
+### 4.4.0-rc1 (Nov 21st 2023)
+
+This release adds important corrections to
+[4.3.0](https://github.com/fossology/fossology/releases/tag/4.3.0)
+
+The release 4.4.0-rc1 introduces a number of corrections to
```

Comment:
```suggestion
The release [4.4.0-rc1](https://github.com/fossology/fossology/releases/tag/4.4.0-rc1) introduces a number of corrections to
```

Reply:
We never provided this link in the changelogs before. Do you want me to include them?